src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -573,9 +573,7 @@ private void ReportExportedTypeNameCollisions(ImmutableArray<Cci.ExportedType> e
 
                     while (stack.Count > 0)
                     {
-                        var entry = stack.Pop();
-                        NamedTypeSymbol type = entry.Item1;
-                        int parentIndex = entry.Item2;
+                        var (type, parentIndex) = stack.Pop();
```

Comment:
Nice
Perhaps use types instead of var: `(NamedTypeSymbol type, int parentIndex) = stack.Pop();`